2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -78,6 +79,16 @@ class CollectorService(
   override val cookieName = config.cookieName
   override val doNotTrackCookie = config.doNotTrackHttpCookie
 
+  /**
+   * Determines the path to be used in the response,
+   * based on whether a mapping can be found in the config for the original request path.
+   */
+  override def determinePath(vendor: String, version: String): String = {
+    val original = s"$vendor/$version"
+    val mappingOrOriginal = config.paths.getOrElse(original, original)
```

Comment:
Is there a reason why we don't have a slash in the beginning in the mapping?

Reply:
Do you mean in the config? I think it's more user-friendly and less error-prone without it. Compare:

```
paths {
  "/com.acme/track" -> "/com.snowplowanalytics.snowplow/tp2
}
```

vs

```
paths {
  "com.acme/track" -> "com.snowplowanalytics.snowplow/tp2
}
```

Reply:
That's arguable. For me `/com.acme/track` looks explicit in the sense that is a root-level URL. Though it is a nit, I don't have strong opinion about it.